iron-iconset-svg.html

```diff
@@ -136,6 +136,10 @@
      */
     _nameChanged: function() {
       new Polymer.IronMeta({type: 'iconset', key: this.name, value: this});
+      this.async(function() {
```

Comment:
So it looks like nothing goes wrong without the async here, save that the whole thing becomes more difficult to test (we can't use fixtures because by the time that we've got a reference to the fixture, the event has already been dispatched!).